scripts/automation_cli/__main__.py

```diff
@@ -0,0 +1,18 @@
+"""Automation scripts CLI."""
```

Comment:
To any python script we should add license header. I know... it's not everywhere, but just to be correct on a new files.

Reply:
like this? (got it from `scripts/python-version-check.py`)
```python
# Copyright Red Hat
#
#    Licensed under the Apache License, Version 2.0 (the "License"); you may
#    not use this file except in compliance with the License. You may obtain
#    a copy of the License at
#
#         http://www.apache.org/licenses/LICENSE-2.0
#
#    Unless required by applicable law or agreed to in writing, software
#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
#    License for the specific language governing permissions and limitations
#    under the License.
#
```
I believe this is not in the documentation. I think it would be good to add this in the Development guide.